modules/components/style.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var hyphenateStyleName = require('react/lib/hyphenateStyleName');
+var reduce = require('lodash.reduce');
+
+function buildCssString (selector, rules) {
+  // TODO: Better error checking for rules object
+  if (!selector || !rules) {
+    return;
+  }
+
```

Comment:
totally!

And for this project, I'm just making suggestions, don't wait up for me ;)